frontend/src/api/api.tsx

```diff
@@ -305,6 +305,11 @@ const patchIntegrationConfiguration = async (
   return response.data as IntegrationConfiguration;
 };
 
+const sendNotification = async (users: RoadmapUser[], task: Task) => {
+  const response = await axios.post(`/notify`, { users, task });
```

Comment:
I'm wondering if the route should follow the pattern of roadmaps/tasks/notify?
```suggestion
  const response = await axios.post(
    `roadmaps/${task.roadmapId}/tasks/${task.id}/notify`,
    { users, task },
  );
```